From: Balint Reczey <balint.reczey@canonical.com>
Date: Mon, 18 Mar 2019 13:13:30 +0100
Subject: Makefile: Honor DESTDIR

---
 Makefile | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/Makefile b/Makefile
index dadbd3e..43647f6 100644
--- a/Makefile
+++ b/Makefile
@@ -28,10 +28,10 @@ link:
 	ln -s $(CURPATH)/src/etc /usr/share/wslu
 
 install:
-	install -m755 out/* /usr/bin
-	install -m555 docs/* /usr/share/man/man1
-	[ -d /usr/share/wslu ] || mkdir -p /usr/share/wslu
-	cp src/etc/* /usr/share/wslu
+	install -m755 out/* $(DESTDIR)/usr/bin
+	install -m555 docs/* $(DESTDIR)/usr/share/man/man1
+	[ -d $(DESTDIR)/usr/share/wslu ] || mkdir -p $(DESTDIR)/usr/share/wslu
+	cp src/etc/* $(DESTDIR)/usr/share/wslu
 
 uninstall: 
 	for f in $(INSTEDEXES); do \
