From: Balint Reczey <balint.reczey@canonical.com>
Date: Wed, 16 Oct 2019 11:43:44 +0200
Subject: Fix setting ~/.config/wslu/baseexec when the Windows user name
 contains space

---
 src/wslu-header | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/wslu-header b/src/wslu-header
index 6f893c6..05533bf 100644
--- a/src/wslu-header
+++ b/src/wslu-header
@@ -120,7 +120,9 @@ if [ ! -f ~/.config/wslu/baseexec ] || [ -z "$(cat ~/.config/wslu/baseexec)" ];
 		wslu_distro_regpath=$("$(interop_prefix)"c/Windows/System32/reg.exe query "HKCU\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Lxss" /s /f DistributionName 2>&1 | grep -B1 -e "$WSL_DISTRO_NAME" | head -n1 | sed -e 's|\r||g')
 		wslu_distro_packagename=$($(interop_prefix)c/Windows/System32/reg.exe query "$wslu_distro_regpath" /v PackageFamilyName | tail -n 2 | head -n 1 | sed -e 's|\r||g')
 		wslu_distro_packagename=${wslu_distro_packagename##* }
-		find $(wslpath "$(double_dash_p $(winps_exec "Write-Output \$Env:LOCALAPPDATA" | tr -d "\r"))\\Microsoft\\WindowsApps\\$wslu_distro_packagename") -name "*.exe" > ~/.config/wslu/baseexec
+                distro_path_win=$(winps_exec 'Write-Output $Env:LOCALAPPDATA' | tr -d '\r')\\Microsoft\\WindowsApps\\$wslu_distro_packagename
+                distro_path=$(wslpath "$distro_path_win")
+		find "$distro_path" -name "*.exe" > ~/.config/wslu/baseexec
 	else
 		echo "$(wslpath "C:\\Windows\\System32\\wsl.exe")" > ~/.config/wslu/baseexec
 	fi
