From: "Jinming Wu, Patrick" <me@patrickwu.space>
Date: Mon, 11 May 2020 19:54:58 +0800
Subject: [PATCH] src/wslfetch.sh: update Ubuntu logo

---
 src/wslfetch.sh | 38 ++++++++++++++++++++------------------
 1 file changed, 20 insertions(+), 18 deletions(-)

diff --git a/src/wslfetch.sh b/src/wslfetch.sh
index 4d0aea4..351df71 100644
--- a/src/wslfetch.sh
+++ b/src/wslfetch.sh
@@ -24,24 +24,26 @@ case "$distro" in
 	'ubuntu')
 		t="${red}${bold}"
 		full_text=(
-			"${red}                          ./+o+-      ${reset}"
-			"${reset}                  yyyyy. ${red}'yyyyyy+     ${reset}"
-			"${reset}              .;//+/////h ${red}yyyyyyo     ${reset}"
-			"${yellow}           .++ ${reset}.:/++++++/-${red}.\`sss/\`     ${reset}"
-			"${yellow}         .:++o: ${reset}\`\++++++++/:---:/-    ${reset}"
-			"${yellow}        o:+o+:++. ${reset}\`\`\`\`\`'-/ooo+++++\   ${reset}"
-			"${yellow}       .:+o:+o/.          ${reset}\`+sssooo+\  ${reset}"
-			"${reset}  .++/+ ${yellow}+oo+o:\`             ${reset}\sssooo;  ${reset}"
-			"${reset} /+++//+: ${yellow}oo+o                        ${reset}"
-			"${reset} \+/+o+++ ${yellow}o++o               ${red}ydddhh+  ${reset}"
-			"${reset}  .++.o+ ${yellow}+oo+:\`             ${red}/dddhhh;  ${reset}"
-			"${yellow}       .+.o+oo:.           ${red}oddhhhh+   ${reset}"
-			"${yellow}        \+.++o+o\` ${red}-,,,,.:ohdhhhhh+    ${reset}"
-			"${yellow}         \`:o+++  ${red}ohhhhhhhhyo++os:     ${reset}"
-			"${yellow}           .o: ${red}.syhhhhhhh'${yellow}.oo++o.     ${reset}"
-			"${red}               /osyyyyyyy${yellow}.oooo+++\    ${reset}"
-			"${red}                   \`\`\`\`\` ${yellow}+oo+++o:/    ${reset}"
-			"${yellow}                          \`oo++'\`     ${reset}");;
+			"${bold}${red}               .-/+oossssoo+/-.               ${reset}"
+			"${bold}${red}           \`:+ssssssssssssssssss+:\`           ${reset}"
+			"${bold}${red}         -+ssssssssssssssssssyyssss+-         ${reset}"
+			"${bold}${red}       .ossssssssssssssssss${white}dMMMNy${red}sssso.       ${reset}"
+			"${bold}${red}      /sssssssssss${white}hdmmNNmmyNMMMMh${red}ssssss/      ${reset}"
+			"${bold}${red}     +sssssssss${white}hm${red}yd${white}MMMMMMMNddddy${red}ssssssss+     ${reset}"
+			"${bold}${red}    /ssssssss${white}hNMMM${red}yh${white}hyyyyhmNMMMNh${red}ssssssss/    ${reset}"
+			"${bold}${red}   .ssssssss${white}dMMMNh${red}ssssssssss${white}hNMMMd${red}ssssssss.   ${reset}"
+			"${bold}${red}   +ssss${white}hhhyNMMNy${red}ssssssssssss${white}yNMMMy${red}sssssss+   ${reset}"
+			"${bold}${red}   oss${white}yNMMMNyMMh${red}ssssssssssssss${white}hmmmh${red}ssssssso   ${reset}"
+			"${bold}${red}   oss${white}yNMMMNyMMh${red}sssssssssssssshmmmh${red}ssssssso   ${reset}"
+			"${bold}${red}   +ssss${white}hhhyNMMNy${red}ssssssssssss${white}yNMMMy${red}sssssss+   ${reset}"
+			"${bold}${red}   .ssssssss${white}dMMMNh${red}ssssssssss${white}hNMMMd${red}ssssssss.   ${reset}"
+			"${bold}${red}    /ssssssss${white}hNMMM${red}yh${white}hyyyyhdNMMMNh${red}ssssssss/    ${reset}"
+			"${bold}${red}     +sssssssss${white}dm${red}yd${white}MMMMMMMMddddy${red}ssssssss+     ${reset}"
+			"${bold}${red}      /sssssssssss${white}hdmNNNNmyNMMMMh${red}ssssss/      ${reset}"
+			"${bold}${red}       .ossssssssssssssssss${white}dMMMNy${red}sssso.       ${reset}"
+			"${bold}${red}         -+sssssssssssssssss${white}yyy${red}ssss+-         ${reset}"
+			"${bold}${red}           \`:+ssssssssssssssssss+:\`           ${reset}"
+			"${bold}${red}               .-/+oossssoo+/-.               ${reset}");;
 	'debian')
 		t="${light_red}${bold}"
 		full_text=(
